src/constants/externalUrls.ts

```diff
@@ -1,6 +1,3 @@
-export const REPORT_PROBLEM_ACCESSIBILITY_TEAM_SURVEY =
-  'https://docs.google.com/forms/d/e/1FAIpQLSfRJIddKkcffFWXB4AVG_ibYgShu1t-HcKuXov06mZDDxwjLw/viewform';
-export const REPORT_PROBLEM_BASIC_USER_SURVEY =
-  'https://www.surveymonkey.com/r/GCYMVY8';
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Is this needed because there's now only a single item, and eslint thinks there should just be a default?

Reply:
I think export default would change the import on the other end. Since this file could have more exports again in the future I thought this line exception was the better thing